nuke-from-orbit/loadtester

```diff
@@ -404,8 +415,17 @@ case "$1" in
   teardown)
     teardown
   ;;
+  update)
+    if [[ $# -ne 2 ]]
+    then
+      echo $"Usage: $0 update [tag] <- a new tag for the updated test case"
```

Comment:
If you can't reuse the same tag name consider making a suggestion here to tack a version number onto the tag since that's what people are going to have to end up doing as they iterate

Reply:
done.